src/components/Display/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ number, changeNumber }) => (
+    <div>
+        <input type="text" value={number} onChange={() => changeNumber(number) } /> 
+    </div>
+);
+
+Display.propTypes = {
+    number: PropTypes.number.isRequired,
+    changeNumber: PropTypes.func.isRequired,
+};
+
+export default Display;
```

Comment:
It's a common practice to left the last line of the file, blank.

Reply:
Sure